packages/wallets/coin98/src/adapter.ts

```diff
@@ -158,7 +158,37 @@ export class Coin98WalletAdapter extends BaseSignerWalletAdapter {
         const signedTransactions: Transaction[] = [];
         for (const transaction of transactions) {
             signedTransactions.push(await this.signTransaction(transaction));
+            await this.sleep()
         }
         return signedTransactions;
     }
+
+
+    async signMessage(message: Uint8Array): Promise<Uint8Array> {
+        try {
+            const wallet = this._wallet;
+            if (!wallet) throw new WalletNotConnectedError();
+
+            try {
+                const decodedMessage = new TextDecoder().decode(message)
```

Comment:
Hmm it would probably be better to standardize your interface around something common like Base64 than using `TextDecoder.decode`'s return value of a [DOMString](https://developer.mozilla.org/en-US/docs/Web/API/DOMString).

Reply:
Oh, base58 might be better, as that's what's used for standard Solana transaction signatures.